packages/client/src/orders/types/orderShippingAddressChangeRequests.types.ts

```diff
@@ -0,0 +1,31 @@
+import type { CheckoutAddress } from '../../types';
+import type { CheckoutOrder } from '../../checkout';
+import type { Order } from './order.types';
+
+export type OrderShippingAddressChangeRequests =
+  Array<OrderShippingAddressChangeRequest>;
+
+export type OrderShippingAddressChangeRequest = {
+  id: string;
+  checkoutOrderId: CheckoutOrder['id'];
+  orderId: Order['id'];
+  status: OrderShippingAddressChangeRequestStatus;
+  createdDate: string;
+  updatedDate: string;
+  merchantOrders: Array<MerchantOrder>;
```

Comment:
According to the endpoint, this is a single object with an id property. Can you confirm, please?

Reply:
Correct, I don't know where I got the array from... Fixed!

Reply:
It is indeed an array, validated by the BE team. Fixed, again.